kombu/utils/url.py

```diff
@@ -38,6 +44,24 @@ def parse_url(url):
     # type: (str) -> Dict
     """Parse URL into mapping of components."""
     scheme, host, port, user, password, path, query = _parse_url(url)
+    if query:
+        keys = list(query.keys())
+        for key in keys:
+            if key.startswith('ssl_'):
+                if key == 'ssl_cert_reqs':
```

Comment:
Why do we need a loop if we're only searching for this key?
We can simply check for it and pop before the loop.

Reply:
it is not one key but also `ssl_ca_certs`, `ssl_certfile`, and `ssl_keyfile`. This is here because `ssl_cert_reqs` is the only key that needs to be modified from a string to a `ssl.` constant.